ops/testing.py

```diff
@@ -977,13 +1011,22 @@ def status_set(self, status, message='', *, is_app=False):
             self._unit_status = {'status': status, 'message': message}
 
     def storage_list(self, name):
-        raise NotImplementedError(self.storage_list)
+        return [id for id, props in self._storage_list[name].items()]
```

Comment:
Why not `[id for id in self._storage_list[name].keys()]` ?

Reply:
Or `list(self._storage_list[name].keys())`?